imagekit-sdk/src/main/java/io/imagekit/sdk/tasks/RestClient.java

```diff
@@ -59,6 +59,9 @@ public Result upload(FileCreateRequest fileCreateRequest){
                 result=new Gson().fromJson(resp,Result.class);
                 result.setSuccessful(true);
                 result.setRaw(resp);
+                if (response.headers()!=null) {
+                	result.setHeaders(response.headers().toMultimap());
```

Comment:
It should be set also in non-success responses.

Reply:
have fixed this and pushed the changes.